mitosheet/src/mito/components/endo/celleditor/cellEditorUtils.tsx

```diff
@@ -41,20 +44,15 @@ export const getSelectionFormulaString = (selections: MitoSelection[], sheetData
     accepted these pending selected columns.
 */
 export const getFullFormula = (
-    formula: string, 
-    pendingSelections: {
-        selections: MitoSelection[],
-        inputSelectionStart: number,
-        inputSelectionEnd: number,
-    } | undefined,
+    editorState: EditorState,
     sheetData: SheetData,
```

Comment:
Ok -- this function call gets really confusing. previously, sheetData was the sheet being edited. Now it's the sheet that happens to be selected at the time of calling this function - which is a weird and hard to express invariant that isn't super clear. 

My feeling: we should make `getFullFormula` take the sheetDataArray, and then pass that down to the `getSelectionFormulaString`. Then, we can get the dfName in this case more clearly -- and it's really obvious what this function is doing / requires. 

It removes the need to remember _which_ sheetData you need to pass to `getFullFormula` -- is it the sheetData of the formula, or the sheetData of the currently selected sheet? I'd rather not think about it.